README.md

```diff
@@ -42,7 +42,7 @@ Check out the [documentation 📚](https://starlite-api.github.io/starlite/).
 pip install starlite
 ```
 
-## Quick Start
```

Comment:
I'd go with consistent heading capitalisation

Reply:
My personal preference is to capitalise every first letter of the words like so - "Quick Start" but [Google suggests otherwise](https://developers.google.com/style/headings#heading-and-title-format) :sweat_smile: 

We can configure Vale to ignore such practices but then it'll defeat the purpose of using a standard linter.

Reply:
+1 for just taking the style guide as is.
Also: `Quickstart` is a very common way to spell this.